tests/public-api/requirements/_base.txt

```diff
@@ -1,5 +1,5 @@
 # This file was autogenerated by uv via the following command:
-#    uv pip compile --output-file requirements/_base.txt requirements/_base.in
+#    uv pip compile --upgrade-package aiodocker --output-file requirements/_base.txt requirements/_base.in
```

Comment:
undo this change. It tracked that you upgraded a single package. It is the correct way to go but we should not upgrade base.txt for this. 
I wonder if uv has some options to avoid these changes otherwise it is going to be a bit more time consuming upgrading packages

Reply:
ok.